src/map_homework/model/Student.java

```diff
@@ -0,0 +1,65 @@
+package map_homework.model;
+
+import java.util.*;
+
+public class Student {
+    private String firstName;
+    private String lastName;
+    private int age;
+    private String phoneNumber;
+    private String faculty;
+
+    public Student(String firstName, String lastName, int age, String phoneNumber, String faculty) {
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.age = age;
+        this.phoneNumber = phoneNumber;
+        this.faculty = faculty;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public String getPhoneNumber() {
+        return phoneNumber;
+    }
+
+    public String getFaculty() {
+        return faculty;
+    }
+
+    @Override
+    public String toString() {
+        return firstName + " " + lastName;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        Student student = (Student) o;
+        return firstName.equals(student.firstName) && lastName.equals(student.lastName);
```

Comment:
Is this equals() method auto-generated or created by yourself? Can you try to avoid NullPointerException?

Reply:
I've copied the equals()  from last homework and a little bit modified it

Reply:
Try to generate it and see the difference.